.travis.yml

```diff
@@ -70,8 +72,25 @@ matrix:
       python: "3.6"
       env: TOXENV=py36-flake8
 
+    - stage: Deploy
+      name: Publishing current Git tagged version of dist to PyPI
+      if: tag IS present
+      install: skip
+      script: skip
+      deploy:
+        provider: pypi
+        user: any-maintainer-on-pypi  # I recommend having a separate account
```

Comment:
@gundalow this will require creating a "service" account on PyPI just for pushing releases.